src/gluonts/mx/block/regularization.py

```diff
@@ -25,9 +25,8 @@
 class ActivationRegularizationLoss(Loss):
     r"""Computes Activation Regularization Loss. (alias: AR)
     The formulation is as below:
-    .. math::
-    L = \alpha L_2(h_t)
-    where :math:`L_2(\cdot) = {||\cdot||}_2, h_t` is the output of the RNN at timestep t.
+    .. math:: L = \alpha L_2(h_t)
+    where :math:`L_2(\cdot) = {||\cdot||}_2^2, h_t` is the output of the RNN at timestep t.
```

Comment:
More directly:

```suggestion
    .. math:: L = \alpha \|h_t\|_2^2, 
    where :math:`h_t` is the output of the RNN at timestep t.
```

@xcgoner you can check that this docstring is rendered as intended by going to the docs/ folder, hitting `make`, and navigating starting from docs/_build/html/index.html